### PR TITLE
Add EKS installation procedure

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+
+.. code-block:: console
+
+   # Logout of helm registry to perform an unauthenticated pull against the public ECR
+   helm registry logout public.ecr.aws
+
+   helm upgrade --install \
+     --namespace <namespace> \
+     --create-namespace \
+     llm-operator oci://public.ecr.aws/cloudnatix/llm-operator-charts/llm-operator \
```
Comment (guangrui-cloudnatix) [inline docs/source/install.rst:14]: should it be `oci://public.ecr.aws/cloudnatix/llmariner-charts/llm-operator`

Comment (kkaneda) [reply docs/source/install.rst:14]: Yeah I'll do all the renaining later.

File: docs/source/install.rst [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+           runtimeName: ollama
+           preloaded: true
+           resources:
+            limits:
+              nvidia.com/gpu: 0
+
+   inference-manager-server:
+     service:
+       annotations:
+         konghq.com/connect-timeout: "360000"
```
Comment (guangrui-cloudnatix) [inline docs/source/install.rst:455]: might want to add a comment that these are needed only if using `kong` for ingress?

**Review by guangrui-cloudnatix — APPROVED** [review]

**Review by Ladicle — APPROVED** [review]

File: docs/source/install.rst [hunk 3]
```diff
@@ -0,0 +1,570 @@
...
+
+LLM Operator can process inference requests on CPU nodes, but it can be best used with GPU nodes. Nvidia GPU Operator is required to install the device plugin and
+make GPUs visible in the K8s cluster.
+
+Preferably the ingress controller should have a DNS name or a IP that is reachable from the outside of the EKS cluster. If not, you can rely on
+port-forwarding to reach the API endpoints.
+
+.. note::
+
+   When port-forwarding is used, the same port needs to be used consistently as the port number will bee included the OIDC issuer URL. We will explain details later.
```
[the remark this answers is not in the document]
Comment (kkaneda) [reply docs/source/install.rst:51]: done!

File: docs/source/install.rst [hunk 4]
```diff
@@ -0,0 +1,570 @@
...
+
+The rest of the section go through concrete steps to create an EKS cluster, create necessary resources, and install LLM Operator. You can
+skip some of the steps if you have already made necessary installation/setup.
+
+
+Step 1. Provision EKS cluster with Karpenter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow the `Karpenter getting started guide <https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/>`_ and
+create a EKS cluster and add Karpenter. The following is the installation step copied from the page (with slight simplification).
```
[the remark this answers is not in the document]
Comment (kkaneda) [reply docs/source/install.rst:81]: done

File: docs/source/install.rst (hunk 2 above)
Comment (kkaneda) [reply docs/source/install.rst:455]: done

File: docs/source/install.rst [hunk 5]
```diff
@@ -0,0 +1,570 @@
...
+   model-manager-loader:
+     serviceAccount:
+       create: false
+       name: "${LLM_OPERATOR_SERVICE_ACCOUNT_NAME}"
+     baseModels:
+     - meta-llama/Meta-Llama-3.1-8B-Instruct-q4_0
+     - google/gemma-2b-it-q4_0
+     - sentence-transformers/all-MiniLM-L6-v2-f16
+
+   vector-store-manager-server:
```
[the remark this answers is not in the document]
Comment (kkaneda) [reply docs/source/install.rst:478]: done

File: docs/source/install.rst [hunk 6]
```diff
@@ -0,0 +1,570 @@
...
+     llmEngineAddr: ollama-sentence-transformers-all-minilm-l6-v2-f16:11434
```
[the remark this answers is not in the document]
Comment (kkaneda) [reply docs/source/install.rst:479]: Let me leave this is as is for now. I think we need to fix this anyway as it is not very intuitive that embedding models are requried.

**Pull request merged**